### ISSUE TITLE
Handle empty image alt text

### ISSUE THREAD

**Issue by halovanic** [mikeedwards83/Glass.Mapper#190]
As of v4.0.5.54 of Glass.Mapper.SC, if you call RenderImage with a Sitecore image that has no ALT text configured, you get back an IMG tag with no ALT attribute rendered.

It would be better if Glass rendered these cases as an empty ALT attribute for [accessibility reasons](http://www.w3.org/TR/WCAG20-TECHS/H67.html), so as to correctly mark up decorative images that should not be read out by screen readers.  Everything I can see in the various accessibility standards indicates that alt attributes should now be considered non-optional.

In GlassHtml.cs, this check is setting the alt text only if it's non-empty:

    AttributeCheck(attributes, ImageParameterKeys.ALT, image.Alt);

This could be replaced with:

    if (attributes[ImageParameterKeys.ALT].IsNullOrEmpty() && !image.Alt.IsNullOrEmpty())
        attributes[ImageParameterKeys.ALT] = image.Alt;
    else
        attributes[ImageParameterKeys.ALT] = string.Empty;

Thanks,
Alex

**Comment by mikeedwards83** [comment]
GlassHtml has been updated to output the alt attribute even when empty

**Comment by halovanic** [comment]
Thank you!

**Comment by mikeedwards83** [comment]
This has been released in 4.0.6.55
